recipes/polly/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "16.0.4" %}
+{% set major_ver = version.split(".")[0] %}
+
+package:
+  name: polly
+  version: {{ version }}
+
+source:
+  url: https://github.com/llvm/llvm-project/releases/download/llvmorg-{{ version.replace(".rc", "-rc") }}/llvm-project-{{ version.replace(".rc", "rc") }}.src.tar.xz
+  sha256: cf3c2a1152ed7a99866bd7f12c24528ada6d9f6336afb7a08416938762004c9f
+
+build:
+  number: 0
+  skip: true  # [win or osx]
```

Comment:
Remove skip for osx ?

Reply:
I tried it, but
- cxx-compilers depends on llvm-tools=15.0.4 for osx
- llvmdev=16.0.4 includes llvm-tools=16.0.4,
so I get a version conflict.

It might be that polly could depend on a subset of llvmdev, but I don't know the details.

Reply:
With the updated requirements (llvmdev in host and not build) this should not be a problem.

Reply:
I see! Cool, will change this later

Reply:
> so I get a version conflict.

That's because we have a default compiler version on osx (clang 15) which conflicts with getting some other required LLVM artefacts in version 16.

As Isuru said, we should be able to avoid that by not mixing the build requirements (where the compiler sits) with the host & run requirements, but even then it would be possible to fix, because you can request a different compiler version through `recipe/conda_build_config.yaml` with something like the following:
```
c_compiler_version:    # [osx]
  - 16                 # [osx]
cxx_compiler_version:  # [osx]
  - 16                 # [osx]
```

Reply:
Should we start with 15 here and then upgrade to 16 in the feedstock?